wls-ergebnismeldung-service/src/test/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/client/eai/EAIClientImplTest.java

```diff
@@ -28,7 +28,7 @@ class EAIClientImplTest {
     @Mock
     WahldatenControllerApi wahldatenControllerApi;
     @Mock
-    WahlergebnisControllerApi wahlergebnisControllerApi;
+    ErgebnismeldungControllerApi wahlergebnisControllerApi;
```

Comment:
```suggestion
    ErgebnismeldungControllerApi ergebnismeldungControllerApi;
```
im zugehörigen file zum test hast du die variable auch umbenannt

Reply:
fixed mit 540c69bb10f081119cd39d0f5f603620643c762f